internals/daemon/daemon.go

```diff
@@ -368,6 +380,22 @@ func (d *Daemon) Init() error {
 		logger.Noticef("HTTP API server listening on %q.", d.httpAddress)
 	}
 
+	if d.httpsAddress != "" {
+		tlsConf := &tls.Config{
+			// We now support HTTP1.1 and HTTP2 when using HTTPS.
+			NextProtos:     []string{"h2", "http/1.1"},
+			MinVersion:     tls.VersionTLS13,
+			MaxVersion:     tls.VersionTLS13,
```

Comment:
I'm definitely not a TLS expert, but why do we want a maximum here?

Reply:
I agree with your feeling. I've removed it :) Resolving.